content/arduino-cloud/09.business/02.aws-s3-exporter/content.md

```diff
@@ -0,0 +1,465 @@
+
+![lambdas3binaries Bucket Setup (1)](assets/s3_bucket_lambdas3bin_setup_1.png)
+
+Select the **Upload** option within the **Objects** panel.
+
+Manually upload the required files by either using the file browser to select the `.zip` and `.yaml` files or dragging and dropping them into the designated upload area. Once the files are recognized, the screen should resemble the following image:
+
+![lambdas3binaries Bucket Setup (2)](assets/s3_bucket_lambdas3bin_setup_2.png)
+
+Click the **Upload** button to begin the upload. Once the upload is complete, a confirmation screen will be displayed:
+
+![lambdas3binaries Bucket Setup (3)](assets/s3_bucket_lambdas3bin_setup_3.png)
+
+After a successful upload, the files will be listed under the **Objects** tab within the **`lambdas3binaries`** bucket:
+
+![lambdas3binaries Bucket Setup (4)](assets/s3_bucket_lambdas3bin_setup_4.png)
+
+#### CSV Destination Bucket
+
+A second bucket needs to be created following the same process as the [Temporary bucket](#temporary-bucket). This bucket will be the **CSV destination bucket**, where all generated CSV files will be stored. It is important to make sure this bucket is created in the same AWS region where the CloudFormation stack will be deployed.
+
+Navigate to the **Amazon S3** service and select **Create bucket**. In the bucket creation window, specify the bucket name and check that the same AWS region is selected.
+
+Keep the recommended default settings for **Object Ownership** and **Public Access** to maintain security compliance. Once all settings are verified, proceed with the bucket creation.
+
+![lambdas3binaries Bucket Setup (1)](assets/s3_bucket_csvdests3int_1.png)
+
+After the bucket has been successfully created, it will be listed among the available S3 buckets. Select the newly created **CSV destination bucket** to proceed with additional configurations if necessary.
+
+![lambdas3binaries Bucket Setup (2)](assets/s3_bucket_csvdests3int_2.png)
+
+Inside the **CSV destination bucket**, navigate to the **Objects** tab. Files can be organized by creating directories if needed.
+
+Click on the **Create folder** button and specify a name for the directory where the exported CSV files will be stored.
+
+![lambdas3binaries Bucket Setup (3)](assets/s3_bucket_csvdests3int_3.png)
+
+During folder creation, options for **server-side encryption** will be displayed for data protection. By default, encryption settings derives from the bucket’s global configuration.
+
+A **custom encryption key** can be specified before finishing the folder creation.
+
+![lambdas3binaries Bucket Setup (4)](assets/s3_bucket_csvdests3int_4.png)
+
+Once the folder is created, it will be displayed under the **Objects** tab of the **CSV destination bucket**. This makes sure that all exported CSV files are well organized within the dedicated bucket.
```

Comment:
```suggestion
Once the folder is created, it will be displayed under the **Objects** tab of the **CSV destination bucket**. This ensures that all exported CSV files are well-organized within the dedicated bucket.
```